### DISCUSSION TITLE
SMS Channel / Provider Integration

### DISCUSSION THREAD

**Discussion by baljeetsingh** [Sunbird-RC/community#278] [Ideas & Enhancements]
Hello,

Would like to know how and where to integrate a sms provider for sending messages through RC.

notify: @coolbung , @parthlawate , @Renuka-S

**Comment by tejash-jl** [comment]
@baljeetsingh,
we have notification_service which is configured to send sms/email. The related config can be set via ENV using [these properties](https://github.com/Sunbird-RC/sunbird-rc-core/blob/main/services/notification-service/config/config.go#L25-L28).

https://github.com/Sunbird-RC/sunbird-rc-core/blob/main/docker-compose.yml#L140-L143

RIght now notifications are configured only for the invite flow. On successfully invite, the owners will be notified

**Reply by baljeetsingh** [reply]
thanks @tejash-jl will check this out .

**Comment by parthlawate** [comment]
Is your usecase general notifications OR OTP login @baljeetsingh ?

**Reply by baljeetsingh** [reply]
@parthlawate to start with it OTP and later general notifications may follow.
